schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/util/FormatUtil.java

```diff
@@ -31,14 +32,18 @@
     private static final ThreadLocal<SimpleDateFormat> START_FORMAT = new ThreadLocal<SimpleDateFormat>() {
         @Override
         protected SimpleDateFormat initialValue() {
-            return new SimpleDateFormat(" [EEE, d MMM HH:mm");
+            SimpleDateFormat sdf = new SimpleDateFormat(" [EEE, d MMM HH:mm");
+            sdf.setTimeZone(TimeZone.getTimeZone("Europe/Zurich"));
+            return sdf;
         }
     };
 
     private static final ThreadLocal<SimpleDateFormat> FINISH_FORMAT = new ThreadLocal<SimpleDateFormat>() {
         @Override
         protected SimpleDateFormat initialValue() {
-            return new SimpleDateFormat(" - HH:mm]");
+            SimpleDateFormat sdf = new SimpleDateFormat(" - HH:mm]");
+            sdf.setTimeZone(TimeZone.getTimeZone("Europe/Zurich"));
+            return sdf;
```

Comment:
Formatting this on the backend is probably wrong...  I think we are even persisting the formatted text right?  This means that if we have a typo, we can only fix the future...  One day...